ios/RNIapIos.m

```diff
@@ -452,4 +454,11 @@ - (NSDictionary *)getPurchaseData:(SKPaymentTransaction *)transaction {
     return [NSString stringWithFormat:@"%p", instance];
 }
 
+static NSString *localeIdentifierToBCP47(NSString* localeIdentifier) 
+{
+  // e.g. `en_US@currency=USD` -> `en-US`.
+  NSString *identifier = [[product.priceLocale.localeIdentifier componentsSeparatedByString:@"@"] objectAtIndex:0];
```

Comment:
The build is failing because the `product` variable is not found. I've already merged this but I found this out later. I'd have to annotate your code before `publishing` until you give us update on this.!

Reply:
Apologies about this. I refactored and missed this.